benches/helpers.rs

```diff
@@ -1,8 +1,12 @@
 use std::time::Duration;
 
-use jsonrpsee::client_transport::ws::{Uri, WsTransportClientBuilder};
-use jsonrpsee::http_client::{HeaderMap, HttpClient, HttpClientBuilder};
-use jsonrpsee::ws_client::{WsClient, WsClientBuilder};
+use jsonrpsee_v0_15::client_transport::ws::{Uri, WsTransportClientBuilder};
+use jsonrpsee_v0_15::http_client::{HttpClient, HttpClientBuilder};
+use jsonrpsee_v0_15::ws_client::{WsClient, WsClientBuilder};
+
+pub use jsonrpsee_v0_15::core::client::{ClientT, SubscriptionClientT};
+pub use jsonrpsee_v0_15::http_client::HeaderMap;
+pub use jsonrpsee_v0_15::rpc_params;
```

Comment:
might be a touch clearer if exposed like eg `use helpers::fixed_jsonrpsee::ClientT` to make it obvious they are from the fixed/old jsonrpsee version?